packages/badge/stories/Badge.stories.tsx

```diff
@@ -0,0 +1,200 @@
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.PRIMARY),
+          borderColor: darken(0.1, colors.PRIMARY),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      secondary: {
+        background: darken(0.1, colors.SECONDARY),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.SECONDARY),
+          borderColor: darken(0.1, colors.SECONDARY),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      success: {
+        background: darken(0.1, colors.SUCCESS),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.SUCCESS),
+          borderColor: darken(0.1, colors.SUCCESS),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      danger: {
+        background: darken(0.1, colors.DANGER),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.DANGER),
+          borderColor: darken(0.1, colors.DANGER),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      warning: {
+        background: darken(0.1, colors.WARNING),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.WARNING),
+          borderColor: darken(0.1, colors.WARNING),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      info: {
+        background: darken(0.1, colors.INFO),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.INFO),
+          borderColor: darken(0.1, colors.INFO),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      dark: {
+        background: darken(0.1, colors.DARK),
+        color: darken(0.1, colors.WHITE),
+        ghost: {
+          color: darken(0.1, colors.DARK),
+          borderColor: darken(0.1, colors.DARK),
+          background: darken(0.1, colors.WHITE),
+        },
+      },
+      light: {
+        background: darken(0.1, colors.LIGHT),
+        color: darken(0.1, colors.CHAROCOAL),
+        ghost: {
+          color: darken(0.1, colors.LIGHT),
+          borderColor: darken(0.1, colors.LIGHT),
+          background: darken(0.1, colors.CHAROCOAL),
+        },
+      },
+    },
+  },
+};
+
+const appearances = [
+  'default',
+  'primary',
+  'secondary',
+  'success',
+  'warning',
+  'danger',
+  'info',
+  'light',
+  'dark',
+];
+
+const Table = (props: any) => (
+  <div style={ { display: 'table', minWidth: '280px' } } { ...props } />
+);
+const Row = (props: any) => (
+  <div style={ { display: 'table-row' } } { ...props } />
+);
+const Cell = (props: any) => (
+  <div style={ { display: 'table-cell', padding: 4 } } { ...props } />
+);
+
+function capitalize(str: string) {
+  // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
+  return str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
Maybe should use `${}` instead plus operator and remove eslint comment?

Reply:
It doesn't matter because is it story =)